app/src/main/java/com/example/tung/moviedb_07/screen/movie_detail/MovieDetailViewModel.java

```diff
@@ -1,9 +1,18 @@
 package com.example.tung.moviedb_07.screen.movie_detail;
 
+import android.content.Context;
```

Comment:
ERROR: Unused import: android.content.Context. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports